src/modules/mavlink/mavlink_main.cpp

```diff
@@ -1784,6 +1784,29 @@ Mavlink::configure_streams_to_default(const char *configure_single_stream)
 		configure_stream_local("WIND_COV", 10.0f);
 		break;
 
+	case MAVLINK_MODE_EV:
+		configure_stream_local("ALTITUDE", 10.0f); // for avoidance
+		configure_stream_local("ATTITUDE", 25.0f);
+		configure_stream_local("ATTITUDE_TARGET", 10.0f);
+		configure_stream_local("CAMERA_TRIGGER", unlimited_rate); // for VIO
+		configure_stream_local("CAMERA_IMAGE_CAPTURED", unlimited_rate); //for VIO
```

Comment:
This should not be needed.

Reply:
we need those topics to enable the realsense D series cameras

Reply:
CAMERA_TRIGGER should be all that is needed. The IMAGE_CAPTURED stream is a much bigger message designed for online geotagging at low rates. (1-5Hz)

Reply:
okay. i see i misunderstood the topic application